docs/js/react.js

```diff
@@ -7364,6 +7364,7 @@ var ReactDOM = mapObject({
   ellipse: 'ellipse',
   g: 'g',
   line: 'line',
+  image: 'image',
```

Comment:
Can you revert the changes to this file, we only update this on official releases.